boule/_sphere.py

```diff
@@ -280,6 +286,24 @@ def reference_normal_gravitational_potential(self):
         """
         return self.geocentric_grav_const / self.radius
 
+    def __str__(self):
+        s = self.name + " - " + self.long_name + "\n"
+        s += "Spheroid:\n"
+        s += f"  Radius: {self.radius} m\n"
+        s += f"  GM: {self.geocentric_grav_const} m³/s²\n"
+        s += f"  Angular velocity: {self.angular_velocity} rad/s"
```

Comment:
Similarly for the Sphere:
```suggestion
        n_spaces = 16
        s = self.name + " - " + self.long_name + "\n"
        s += "Spheroid:\n"
        s += f"  • {'Radius':>{n_spaces}}: {self.radius} m\n"
        s += f"  • {'GM':>{n_spaces}}: {self.geocentric_grav_const} m³/s²\n"
        s += f"  • {'Angular velocity':>{n_spaces}}: {self.angular_velocity} rad/s"
```